keps/0034-instance-health.md

```diff
@@ -106,10 +104,16 @@ From all these resources it would compute readiness based on `Condition: Ready`,
 
 For operators installing their app via e.g. `Deployment`, there’s no need at this point to also check the Status of the underlying Pods because Deployment `Status` already mirrors the readiness of the underlying Pods. This is true to all higher-level types like StatefulSet etc.
 
+To mitigate situation when a plan adds a resource we don't want to include in `Ready` condition, it's possible to opt-out from this when adding annotation `kudo.dev/allow-readiness-computation:false`
```

Comment:
Hmmm... I'm not sure `allow-readiness-computation` is the right way... This feels like we would have a default value of "true" if this annotation is not present, which feels odd to me. I would prefer something that defaults to "false" if it's not there:

What about `ignore-readiness-computation:true` or `skip-for-readiness-computation:true`?

Reply:
I don't mind the default being true but I'll switch to your suggestion :) I like both